ProcessLib/ComponentTransport/ComponentTransportProcessData.h

```diff
@@ -32,19 +31,13 @@ namespace ComponentTransport
 struct ComponentTransportProcessData
 {
     ComponentTransportProcessData(
-        MaterialLib::PorousMedium::PorousMediaProperties&&
-            porous_media_properties_,
         ParameterLib::Parameter<double> const& fluid_reference_density_,
-        std::unique_ptr<MaterialLib::Fluid::FluidProperties>&&
-            fluid_properties_,
         std::unique_ptr<MaterialPropertyLib::MaterialSpatialDistributionMap>&&
             media_map_,
         ParameterLib::Parameter<double> const& decay_rate_,
         Eigen::VectorXd const& specific_body_force_, bool const has_gravity_,
         bool const non_advective_form_)
-        : porous_media_properties(std::move(porous_media_properties_)),
-          fluid_reference_density(fluid_reference_density_),
-          fluid_properties(std::move(fluid_properties_)),
+        : fluid_reference_density(fluid_reference_density_),
           media_map(std::move(media_map_)),
           decay_rate(decay_rate_),
```

Comment:
Shouldn't the decay rate also be handled as MPL property?

Reply:
Technically viable. I am not sure it proper to consider decay rate as part of material physical properties.

Reply:
In my opinion decay should not be consider as material property as this property is specific of radioisotopes or what we consider "components".

Reply:
If I remember correctly, the decay rate was used for tracers too.

Reply:
Whether the decay rate goes into MPL or not, depends on where and how it is used. I didn't look into the details, so no suggestions for now.

Reply:
When the decay rate is a property of a component we should add it as a MPL component .

Reply:
Now it turns clear to me. Decay rate should be under the subtree of aqueous liquid phase and assigned as component property.

Reply:
> If I remember correctly, the decay rate was used for tracers too.

If tracers are not considered components, then decay it is also relevant for tracers.

Reply:
Decay rate invoked from MPL.